src/modules/public/alert/sagas/handleAlertSaga.ts

```diff
@@ -35,10 +36,16 @@ export function* handleAlertSaga(action: AlertPush) {
                         action.payload.message.indexOf('authz.csrf_token_mismatch') > -1) {
                             yield call(callAlertData, action);
                         } else {
-                            yield call(callAlertData, action);
-                            break;
+                            const user = yield select(selectUserInfo);
+
+                            if (!user.email.length && action.payload.message.indexOf('authz.invalid_session') > -1) {
```

Comment:
why do we need this "if" if we just break?

Reply:
@VladyslavP Yes, we do, because user will see the errors 'authz.invalid_session' even when user is not logged in. This "if" prevents such situation